webapp/channels/src/utils/a11y_utils.ts

```diff
@@ -60,3 +60,32 @@ export function focusElement(
         }, 0);
     }
 }
+
+/**
+ * Returns the first focusable child within a given container element,
+ * or null if none is found.
+ *
+ * Focusable elements generally include:
+ *  - <a href="...">
+ *  - <button>, <input>, <select>, <textarea> (unless disabled)
+ *  - Elements with a non-negative tabindex.
+ */
+export function getFirstFocusableChild(container: HTMLElement): HTMLElement | null {
+    if (!container) {
+        return null;
+    }
+
+    // Common selectors for focusable elements:
+    const focusableSelectors = [
+        'a[href]',
```

Comment:
Should this also check if the element is disabled as well?

Reply:
ok, for this one anchor tags (a[href]) don't natively support the disabled attribute nor we use the `aria-disabled="true"` in our code base to identify, (should we for non form elements with tab-index) so, no need to extend this validation.

Reply:
Huh, I didn't realize anchor tags couldn't be disabled. That seems odd, but I assume there's a good reason that the HTML spec doesn't allow that